keyboards/massdrop/thekey/readme.md

```diff
@@ -0,0 +1,20 @@
+# The Key
+
+![The Key](https://massdrop-s3.imgix.net/img_thread/1632776012171.401382764724844340547932-2968copypdp.png?auto=format&fm=jpg&fit=min&w=796&dpr=2&q=35)
```

Comment:
This should be rehosted on imgur or the like.  Something that is less prone to being purged at some point.

Reply:
Moved to imgur link